manifests/n/NIOSH/ARMPS/6.2.02/NIOSH.ARMPS.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NIOSH.ARMPS
+PackageVersion: 6.2.02
+PackageLocale: en-US
+Publisher: National Institute for Occupational Safety and Health
+PackageName: Analysis of Retreat Mining Pillar Stability

```

```suggestion
PackageName: ARMPS
```
This must match Control Panel.